spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -102,6 +104,21 @@
 _PREALLOC_NAME = 'MemoryPreAllocatedResources'
 
 
+def get_front_end_common_pacman_xml_paths():
```

Comment:
I do not like this approach at all!

Would it not be better to have both interface_functions and report_functions each report their own xml file.

Dont have this method at all!

Instead have 
_create_xml_paths
call each of the two functions and directly use the result in 
xml_paths.extend(

Reply:
Note: I believe this has simply been moved to avoid a circular dependency